analog.css

```diff
@@ -0,0 +1,46 @@
+.img{
+    margin-block-start: 10px;
+    width: 300px;
+    height: 300px;
+}
+
+#seconds{
+    height: 130px;
+    width: 2px;
+    background-color: rgb(255, 0, 0);
+    position: relative;
+    left: 951px;
+    bottom: 281px;
```

Comment:
Vezi că limbile nu sunt centrate. În locul tău, aș face un element care să conțină cadranul și limbile. Acelui element i-aș pune `position: relative`, pentru ca limbilor să le dau `position: absolute`